includes/admin/class-sensei-learner-management.php

```diff
@@ -79,13 +79,15 @@ public function __construct( $file ) {
 		$this->page_slug      = 'sensei_learners';
 		$this->menu_post_type = 'course';
 
+		$this->bulk_actions_controller = new Sensei_Learners_Admin_Bulk_Actions_Controller( $this );
```

Comment:
> However, it is incorrect to leave object properties uninitialized after constructor.

Exactly my thought. 👍 The constructor is not registering hooks if not in the admin so I've decided it would be a nice change. Also, this removes the need to always check `is_admin()` before accessing the object.